modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/provider/PowwowServiceProviderUtil.java

```diff
@@ -138,138 +127,112 @@ public static String getJoinPowwowMeetingURL(
 	public static boolean getOptionAutoStartVideo(long powwowMeetingId)
 		throws PortalException {
 
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			powwowMeetingId);
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
 
 		return powwowServiceProvider.getOptionAutoStartVideo(powwowMeetingId);
 	}
 
 	public static String getOptionPassword(long powwowMeetingId)
 		throws PortalException {
 
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			powwowMeetingId);
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
 
 		return powwowServiceProvider.getOptionPassword(powwowMeetingId);
 	}
 
-	public static long getPowwowServerId(String providerType) {
-		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider(
-			providerType);
+	public static long getPowwowServerId() {
+		PowwowServiceProvider powwowServiceProvider = getPowwowServiceProvider();
+		String providerType = PortletProps.get(PortletPropsKeys.POWWOW_PROVIDER_TYPES);
```

Comment:
we may remove `POWWOW_PROVIDER_TYPES` and corresponding property

Reply:
updated